homeassistant/components/lyric/climate.py

```diff
@@ -179,6 +179,11 @@ def __init__(
         ):
             self._attr_hvac_modes.append(HVACMode.HEAT_COOL)
 
+        # Setup supported fan modes
+        self._attr_fan_modes = device.settings.attributes.get("fan", {}).get(
+            "allowedModes", []
+        )
```

Comment:
```suggestion
        self._attr_fan_modes = device.settings.attributes.get("fan", {}).get(
            "allowedModes"
        )
```
We should only set this attribute if we enable the feature